src/frontend/context/SøknadContext.tsx

```diff
@@ -0,0 +1,237 @@
+import React, { createContext, useContext } from 'react';
+
+import { useNavigate } from 'react-router';
+
+import type { Avhengigheter, FeiloppsummeringFeil, ISkjema } from '@navikt/familie-skjema';
+import { feil, ok, useFelt, useSkjema } from '@navikt/familie-skjema';
+import type { Ressurs } from '@navikt/familie-typer';
+import { RessursStatus } from '@navikt/familie-typer';
+
+import { useBehandling } from './behandlingContext/BehandlingContext';
+import useDeepEffect from '../hooks/useDeepEffect';
+import useSakOgBehandlingParams from '../hooks/useSakOgBehandlingParams';
+import { useFagsakContext } from '../sider/Fagsak/FagsakContext';
+import type { IBehandling } from '../typer/behandling';
+import { BehandlingUnderkategori } from '../typer/behandlingstema';
+import type { IForelderBarnRelasjon } from '../typer/person';
+import { ForelderBarnRelasjonRolle } from '../typer/person';
+import type {
+    IBarnMedOpplysninger,
+    IBarnMedOpplysningerBackend,
+    IRestRegistrerSøknad,
+    Målform,
+} from '../typer/søknad';
+import { hentBarnMedLøpendeUtbetaling } from '../utils/fagsak';
+
+interface Props extends React.PropsWithChildren {
+    åpenBehandling: IBehandling;
+}
+
+interface SøknadSkjema {
+    underkategori: BehandlingUnderkategori;
+    barnaMedOpplysninger: IBarnMedOpplysninger[];
+    endringAvOpplysningerBegrunnelse: string;
+    målform: Målform | undefined;
+}
+
+interface SøknadContextValue {
+    barnMedLøpendeUtbetaling: Set<string>;
+    hentFeilTilOppsummering: () => FeiloppsummeringFeil[];
+    nesteAction: (bekreftEndringerViaFrontend: boolean) => void;
+    settVisBekreftModal: (vis: boolean) => void;
+    skjema: ISkjema<SøknadSkjema, IBehandling>;
+    søknadErLastetFraBackend: boolean;
+    visBekreftModal: boolean;
+}
+
+const SøknadContext = createContext<SøknadContextValue | undefined>(undefined);
+
+export const SøknadProvider = ({ åpenBehandling, children }: Props) => {
+    const { vurderErLesevisning, settÅpenBehandling, gjelderInstitusjon, gjelderEnsligMindreårig } =
```

Comment:
Jeg antar at endringene her kun er de standarde for å skrive oss vekk fra constate. Om det er andre endringer er det vanskelig å se, siden git ikke gjenkjenner at denne fila er en renamet versjon av den gamle .ts-fila og dermed ikke klarer diff. Si ifra om jeg skal gå gjennom denne fila grundigere 😊

Reply:
Kun standard endringer ja 🙌